src/main/java/uk/gov/ons/ctp/integration/mockcaseapiservice/CasesConfig.java

```diff
@@ -82,6 +88,17 @@ public void addOrReplaceData(final List<CaseContainerDTO> caseList) throws CTPEx
     }
   }
 
+  /**
+   * add or replace CCS data in the ccs case maps from a list of Cases
+   *
+   * @param ccsCaseList - list of ccs cases
+   */
+  public void addOrReplaceCcsData(final List<CaseContainerDTO> ccsCaseList) throws CTPException {
```

Comment:
No need to throw CTPException here - it's not needed

Reply:
done